pkg/openstack/openstack.go

```diff
@@ -96,7 +96,6 @@ type LoadBalancerOpts struct {
 	MonitorTimeout        util.MyDuration     `gcfg:"monitor-timeout"`
 	MonitorMaxRetries     uint                `gcfg:"monitor-max-retries"`
 	ManageSecurityGroups  bool                `gcfg:"manage-security-groups"`
-	NodeSecurityGroupIDs  []string            // Do not specify, get it automatically when enable manage-security-groups. TODO(FengyunPan): move it into cache
```

Comment:
I should point out that even though this has no `gcfg` tag and it's clearly intended to be deprecated, I'm pretty sure it is still possible for it to have been specified by the user in the config file using the name `nodesecuritygroupids`.

See https://pkg.go.dev/gopkg.in/gcfg.v1?utm_source=godoc#hdr-Data_structure for gcfg

See https://github.com/openshift/cloud-provider-openstack/blob/71c3849fcc6ea272c78b3fb0268d0f091428163f/pkg/openstack/openstack.go#L177-L178 for where we read this value from a file.